source/content/decoupled-sites.md

```diff
@@ -17,31 +17,32 @@ This page refers to products and features which are currently in development or
 
 ## Pantheon's Decoupled Architecture
 
-Pantheon is enabling developers and agencies to easily create and manage decoupled sites from a single platform, by providing two new products: Decoupled Bridge and Edge Sites.
+Pantheon is enabling developers and agencies to easily create and manage decoupled sites from a single platform.
 
 <Enablement link="https://pantheon.io/decoupled-cms?docs" title="Sign up for updates!">
 
-Sign up for Edge Sites updates, or talk to Sales to participate in Early Access for Decoupled Bridge. Click the link above to get started!
+Sign up for Edge Sites updates, or talk to Sales to participate in Early Access for Decoupled Sites. Click the link above to get started!
 
 </Enablement>
 
-### Decoupled Bridge
+### Decoupled Sites
 
-**Decoupled Bridge** is a product for running a [Node.js](https://nodejs.org/) codebase in parallel with a WordPress or Drupal site.
+**Decoupled Sites** run a [Node.js](https://nodejs.org/) codebase in parallel with a WordPress or Drupal site.
 It serves a use case where a domain's public traffic is delivered from a Server-Side Rendered front-end framework such as [Frontity](https://frontity.org/), [Next.js](https://nextjs.org/), or [Nuxt.js](https://nuxtjs.org/) and administrative content editing is done in a WordPress or Drupal interface.
```

Comment:
This better would better reflect our current focus framework wise:

It serves a use case where a domain's public traffic is delivered from a Server-Side Rendered front-end framework [Next.js](https://nextjs.org/), or [Gatsby](https://www.gatsbyjs.com/) and administrative content editing is done in a WordPress or Drupal interface.